<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OAISettingsWindow</class>
 <widget class="QMainWindow" name="OAISettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>349</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Novo Ponto de Coleta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Novo ponto de Coleta OAI-PMH</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="labelOAIAddress">
        <property name="text">
         <string>Endereço OAI:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOAIAddress"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMetadataType">
        <property name="text">
         <string>Padrão de metadados:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditMetadataType"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelXSLT">
        <property name="text">
         <string>Folha de estilo:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KUrlRequester" name="kurlrequesterXSLT"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelScript">
        <property name="text">
         <string>Script externo:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KUrlRequester" name="kurlrequesterScript"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelScriptParameters">
        <property name="text">
         <string>Parâmetros para o script:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditScriptParameters"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="text">
         <string>Nome:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditName"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelInfo">
      <property name="text">
       <string>Dúvidas, consulte a documentação em:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="KUrlLabel" name="kurllabelDocumentation">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string>https://github.com/killertux/capc</string>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="useCursor">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonCancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="text">
         <string>Salvar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>439</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>lineEditOAIAddress</tabstop>
  <tabstop>lineEditMetadataType</tabstop>
  <tabstop>kurlrequesterXSLT</tabstop>
  <tabstop>kurlrequesterScript</tabstop>
  <tabstop>lineEditScriptParameters</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
